### PR TITLE
deltachat-desktop: unstable-2021-08-04 -> 1.21.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
https://github.com/deltachat/deltachat-desktop/blob/v1.21.0/CHANGELOG.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix [hunk 1]
```diff
@@ -14,33 +14,19 @@
 }:
 
 let
-  libdeltachat' = libdeltachat.overrideAttrs (old: rec {
```
Comment (Mic92) [inline pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:17]: Do we now use the binary version provided by deltachat?

File: pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix [hunk 2]
```diff
@@ -52,7 +38,7 @@ in nodePackages.deltachat-desktop.override rec {
   ];
 
   buildInputs = [
-    libdeltachat'
+    libdeltachat
```
Comment (Mic92) [inline pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:41]: Ah, got it. we use it from somewhere else.

**Comment by Mic92** [comment]
oh, no! I was about to merge it and than there is a merge conflict!

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by dotlambda** [comment]
> oh, no! I was about to merge it and than there is a merge conflict!

Always happens when updating `nodePackages`. Should be fixed now.

File: pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix:17]: That version would only work on x86_64-linux and x86_64-darwin.

**Pull request merged**